src/main/java/team/chisel/ctm/client/util/ConnectionLocations.java

```diff
@@ -67,19 +67,15 @@
     private final @Nullable Dir dir;
     private boolean offset;
 
-    private ConnectionLocations(@Nullable Dir dir) {
+    ConnectionLocations(@Nullable Dir dir) {
```

Comment:
Why were these made package-private?

Reply:
Enum constructors are always private. This was change was made to remove the unnecessary modifier.